tests/queries/0_stateless/02479_mysql_connect_to_self.sql

```diff
@@ -1,3 +1,4 @@
 -- Tags: no-fasttest
 SELECT *
 FROM mysql('127.0.0.1:9004', system, one, 'default', '')
+SETTINGS send_logs_level = 'fatal'; -- failed connection tries are ok, if it succeeded after retry.
```

Comment:
BTW I haven't seen the original issue, but https://github.com/ClickHouse/ClickHouse/pull/65917 could be a real fix for it